core/modules/util/testEventThread.cc

```diff
@@ -163,10 +163,10 @@ BOOST_AUTO_TEST_CASE(EventThreadTest) {
         std::mutex goCVMtx;
         // Create more threads than can fit in the pool and don't let any complete.
         // Since they all leave the pool (via peThread->leavePool();), they should all run.
-        int threadsRunning = 2*sz;
+        int threadsRunning = sz*2;
```

Comment:
Would this double the maximum number of threads (right now it's `10000`) specified in the worker's configuration file?

Reply:
This is a unit test, a pretty complicated one at that, but it should have no effect on the czar or workers.